sdk/ai/azure-ai-inference/README.md

```diff
@@ -145,14 +145,84 @@ client.completeStream(new ChatCompletionsOptions(chatMessages))
 
 To compute tokens in streaming chat completions, see sample [Streaming Chat Completions][sample_get_chat_completions_streaming].
 
-<!--
+### Chat with image URL
+
+```java readme-sample-chatWithImageUrl
+List<ChatMessageContentItem> contentItems = new ArrayList<>();
+contentItems.add(new ChatMessageTextContentItem("Describe the image."));
+contentItems.add(new ChatMessageImageContentItem(
+    new ChatMessageImageUrl("<URL>")));
+
+List<ChatRequestMessage> chatMessages = new ArrayList<>();
+chatMessages.add(new ChatRequestSystemMessage("You are a helpful assistant."));
+chatMessages.add(ChatRequestUserMessage.fromContentItems(contentItems));
+
+ChatCompletions completions = client.complete(new ChatCompletionsOptions(chatMessages));
+
+for (ChatChoice choice : completions.getChoices()) {
```

Comment:
You can remove the choices array from all the samples, and directly print from the first choice. We only have one "choice"...

Reply:
@dargilco I added a getChoice() API to ChatCompletions and StreamingChatResponseMessageUpdate and updated the samples and tests to use that instead of `getChoices().get(0)`